aether-entity-extraction-module/extractor/settings.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
+#
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import logging
+
+
+def get_required(name):
+    try:
+        return os.environ[name]
+    except KeyError as key:
+        raise RuntimeError(f'Missing {key} environment variable!')
+
+
+DEBUG = bool(os.environ.get('DEBUG'))
+TESTING = bool(os.environ.get('TESTING'))
+SECRET_KEY = get_required('DJANGO_SECRET_KEY')
```

Comment:
is this a django app?

Reply:
No it is not, but because I have to use feats from django-sdk, it had to be partly setup as a django app, thus the need for a solely python library 😢

Reply:
As far as I check it, you are only using your `TaskHelper` class and the `aether.sdk.utils.request` method (actually a wrapper of `requests.request`). Does it worth? Why don't create the `TaskHelper` class here and use the `requests` library directly?

Reply:
`TaskHelper` class is also used in kernel https://github.com/eHealthAfrica/aether/pull/649/files#diff-1910d7e6a2668bff54e163b34f442549R23